packages/nova-react/src/graphql/hooks.ts

```diff
@@ -134,10 +134,23 @@ export function useLazyLoadQuery<TQuery extends OperationType>(
  *                    fragment.
  * @returns The data corresponding to the field selections.
  */
+export function useFragment(
+  fragmentInput: GraphQLTaggedNode,
+  fragmentRef: null | undefined, // This overload makes sure we do not get `unknown` as the return type.
```

Comment:
This comment shouldn't be necessary, as long as there is a test that would fail without this parameter missing?